cephalopod/webinterface/auth.py

```diff
@@ -1,11 +1,19 @@
-from flask import Blueprint
+from flask import Blueprint, render_template
 
+from cephalopod.menu import menu, breadcrumb
```

Comment:
Can we keep relative imports for now until we change all of them to absolute ones? Mixing both is quite ugly